packages/cln_plugin/lib/src/rpc_method/rpc_command.dart

```diff
@@ -1,46 +1,13 @@
 import 'package:cln_plugin/cln_plugin.dart';
 
 class RPCCommand {
-  /// Method name
-  String name;
```

Comment:
I think the only issue we may run into keeping this field would be related to RPCNotifications, whose only field is `method`, but I think we can use `name` instead.

Reply:
I think we can ignore this one.